lib/ash/action_input.ex

```diff
@@ -63,6 +63,10 @@ defmodule Ash.ActionInput do
       doc: "Context to set on the action input.",
       default: %{}
     ],
+    params: [
```

Comment:
This should be fixed in ash_phoenix.

Reply:
Alright, reverted it in both PR's. 👍

Reply:
But wait, wasn't this change in ash? 🤔 Running spark in ash phoenix and nothing happens.

Reply:
(this)[https://github.com/ash-project/ash/pull/2016/files#diff-297da735f2a1aa9a8298f9cd6c4677289999558616ace20a960e2e481ee641bd]

Reply:
omg, I thought you were talking about spark 🤦

Reply:
What I mean is that this entire fix should not be done on this package. You added a params option here but nothing handles it. The bug you're fixing is the ash_phoenix params one right?

Reply:
https://github.com/ash-project/ash_phoenix/pull/359